Cargo.toml

```
@@ -15,5 +15,5 @@ default-features = false # Disable features which are enabled by default
 features = ["user-hooks"]
 
 [lib]
-name = "tmp_lib"
-path = "src/tmp_lib.rs"
+path = "src/lib.rs"
+name = "lib_template_rust"
```

```suggestion
name = "lib_template_rust"
path = "src/lib.rs"
```

Let's keep things consistent.

---

Fixed in https://github.com/Arrow-air/template-rust-lib/pull/1/commits/10f99ba487d694fce868138e25b71987138371f9